### ISSUE TITLE
The deltas for test users seem wrong in matches/ratings page

### ISSUE THREAD

**Issue by jauggy** [beyond-all-reason/teiserver#262]
Here's what it looks like in production (which is correct)
![1](https://github.com/beyond-all-reason/teiserver/assets/1305569/bdb9b4b4-4ca5-4faa-82bb-3b3173d334d1)

my skill went from 22.13 to 23.0 which is a gain of 0.87

But in my dev environment (wrong deltas; ignore the lack of icons which is due to my fontawesome)
![2](https://github.com/beyond-all-reason/teiserver/assets/1305569/462f647c-f787-43ba-a62f-84674a19f234)

My skill went from 26.29 to 25.13 which is loss of 1.16 but it shows a gain of 1.18

The deltas are also wrong for both skill, uncertainty, and leaderboard rating. I noticed this bug when I was checking Exeler's last PR to add uncertainty to this page. But then I went back to an older state of master and this bug is still there. Something wrong with fake data creation perhaps?

**Comment by jauggy** [comment]
@Jordan-Cottle can you confirm that you're also getting mathematically incorrect deltas on the match ratings page?

**Comment by Jordan-Cottle** [comment]
I noticed that some of the test data for the user did have incorrect deltas. The data in this table comes straight from the database tables, so it seems entirely reasonable that the pre-populated data from the script could just have stuff hard coded that isn't possible to have happen in prod with real data.

Example data from the database:
![image](https://github.com/beyond-all-reason/teiserver/assets/37338122/5c1376c4-473f-46bb-a496-b6c374e66a9b)

Notice it's just json stored in the `value` column, so really any data could be put in there with no correctness guarentee

**Comment by StanczakDominik** [comment]
Yeah, the fake data creation iirc basically uses random uncorrelated numbers for both the deltas and the ratings! So they don't make sense. Good find.

One way to improve this would be to use random numbers for the deltas only, and generate the ratings themselves with a cumulative sum over the deltas.
